.rubocop.yml

```diff
@@ -62,7 +62,7 @@ Style/LeadingCommentSpace:
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#hash-space'
   Enabled: false
 
-Style/MethodCallParentheses:
+Style/MethodCallWithoutArgsParentheses:
```

Comment:
Changes to this file won't be necessary after the previous `rubocop` version is used.

Reply:
will remove that.